app/src/main/java/org/wikipedia/edits/EditsTask.kt

```diff
@@ -1,12 +1,13 @@
-package org.wikipedia.suggestededits
+package org.wikipedia.edits
 
 import androidx.annotation.DrawableRes
 
-class SuggestedEditsTask {
+class EditsTask {
```

Comment:
Generally speaking, classes that have to do with "suggesting" edits to the user should probably keep the terminology.